Library/AppEnvironment.swift

```diff
@@ -241,6 +247,7 @@ public struct AppEnvironment: AppEnvironmentType {
     optimizelyClient: OptimizelyClientType? = AppEnvironment.current.optimizelyClient,
     pushRegistrationType: PushRegistrationType.Type = AppEnvironment.current.pushRegistrationType,
     reachability: SignalProducer<Reachability, Never> = AppEnvironment.current.reachability,
+    remoteConfigClient: RemoteConfigClientType? = AppEnvironment.current.remoteConfigClient,
```

Comment:
remember to update `pushEnvironment` too (only way I knew was looking at the old `optimizelyClient`) in this file on an older branch.